timer.go

```diff
@@ -0,0 +1,42 @@
+package libtime
+
+import (
+	"time"
+)
+
+// StopFunc is used to stop a time.Timer.
+type StopFunc func()
+
+// SafeTimer creates a time.Timer and a StopFunc, forcing the caller to deal
+// with the potential goroutine leak. Encourages safe use of a time.Timer in a
+// select statement, but without the overhead of a context.Context.
```

Comment:
Not calling `Stop` is a resource leak, but is not a goroutine leak in the current implementation. Could you correct the wording?

Reply:
fixed